README.md

```diff
@@ -43,6 +43,17 @@ update.
 * Please note that the date shown is the date that the change was released to the Sandbox or Production.
 
 ---
+
+### 7 February 2024
+
+The following change is now available in Sandbox:
+
+#### self-assessment-accounts-api
+
+New endpoint `Opt In to Coding Out`.
+
+* This endpoint enables a customer (identified by a National Insurance number) to opt in to coding out for a specified tax year.
+
```

Comment:
Can we also and a "---" separator here. Can you also add for the entry below this please. It's been missed.